### PR TITLE
Fix for `CFIEntry.get_decoded()`

### PR DESCRIPTION
**Pull request by dimas3452**
* add test that detects an error
* fixup

### CODE REVIEW

File: test/test_callframe.py [hunk 1]
```diff
@@ -145,6 +147,13 @@ def test_describe_CFI_instructions(self):
             (   '  DW_CFA_def_cfa: r7 (edi) ofs 2\n' +
                 '  DW_CFA_expression: r2 (edx) (DW_OP_addr: 201; DW_OP_deref; DW_OP_deref)\n'))
 
+    def test_CFIEntry_get_decoded(self):
+        test_dir = join('test', 'testfiles_for_unittests')
+        with open(join(test_dir, 'simple_mipsel.elf'), 'rb') as f:
+            elf = ELFFile(f)
+            di = elf.get_dwarf_info()
+            entries = di.CFI_entries()
+            entries[0].get_decoded()
```
Comment (eliben) [inline test/test_callframe.py]: Can you extend this a bit to extract some useful information from get_decoded and test it for some expected value?

**Comment by dimas3452** [comment]
I added some expected value, but this test detects that there is an error in `get_decoded`.
The next commit fixes this.

**Comment by dimas3452** [comment]
OK. I swapped the commits.

**Comment by dimas3452** [comment]
The tests pass on Travis.

**Pull request merged**